test/Mono.Linker.Tests.Cases/DataFlow/PropertyDataFlow.cs

```diff
@@ -292,6 +292,8 @@ public void TestPropertyWithExistingAttributes ()
 			[CompilerGenerated]
 			Type PropertyWithExistingAttributes_Field;
 
+			[ExpectedWarning ("IL2043", "PropertyWithExistingAttributes", "PropertyWithExistingAttributes.get", ProducedBy = ProducedBy.Analyzer)]
+			[ExpectedWarning ("IL2043", "PropertyWithExistingAttributes", "PropertyWithExistingAttributes.set", ProducedBy = ProducedBy.Analyzer)]
 			[DynamicallyAccessedMembers (DynamicallyAccessedMemberTypes.PublicConstructors)]
 			Type PropertyWithExistingAttributes {
 				// Analyzer doesn't try to detect backing fields of properties: https://github.com/dotnet/linker/issues/2273
```

Comment:
This comment is actually wrong - the IL2043 is not about backing fields - the difference is now solely about https://github.com/dotnet/linker/issues/2654. Could you please update the comments?